src/00_hello.py

```diff
@@ -1 +1,42 @@
-# Print "Hello, world!" to your terminal
\ No newline at end of file
+# Print "Hello, world!" to your terminal
+print('Hello world')
+
+##stretch goal to find if a number is prime
+x = input("Enter an integer to see if it is prime: ")
+
+
+def isItPrime(n):
+    i = 2
+    while i <= n:
+        if i == n:
+            print(f"{n} is prime!")
+            return True
+        elif n % i == 0:
+            print(f"{n} is not prime! it has a lowest demoninator of {i}")
+            return False
+        else:
+            i += 1
+
+isItPrime(int(x))
+
+
+
+##implement the sieve of Eratosthenes
+
+def isItPrimeSieve(n):
+    prime = [True for i in range(n + 1)]
```

Comment:
I like that you're applying the comprehensions. FYI though, python offers an even shorter way of creating simple lists of a certain length. The multiplication operator works on them. So, prime = [True] * n is totally valid python.